cpp_linter/run.py

```diff
@@ -84,7 +84,8 @@
     "--version",
     default="10",
     help="The desired version of the clang tools to use. Accepted options are strings "
-    "which can be 6.0, 7, 8, 9, 10, 11, 12. Defaults to %(default)s.",
+    "which can be 6.0, 7, 8, 9, 10, 11, 12. Defaults to %(default)s. On Windows, this "
```

Comment:
```suggestions
"which can be 8, 9, 10, 11, 12, 13, 14. Defaults to %(default)s. On Windows, this "
```

Reply:
https://github.com/shenxianpeng/cpp-linter-action/blob/b641546bb1325301fa34ec731041f6343a5ccd8f/cpp_linter/run.py#L83

This line is an uppercase V, not a lowercase v is that what you expect?